src/components/TabletApp/RightSide/AdvancedBook/AdvancedBook.tsx

```diff
@@ -16,18 +16,15 @@ import {
     Participant,
 } from "../../../../interfaces/interfaces";
 import CONSTANTS from "../../../../constants/Constants";
-
-interface iAdvancedBook {
-    availability: number;
-}
+import { iAdvancedBook } from "../../../../interfaces/interfaces";
 
 const AdvancedBook = ({ availability }: iAdvancedBook) => {
     const [validForm, setValidForm] = useState<FormValidity>({
         isNameValid: false,
         isDateValid: false,
         isStartValid: false,
         isEndValid: false,
-        isOwnerValid: false,
+        isOwnerValid: true,
```

Comment:
Remove this because we only changed it for testing

Reply:
changed it back